### DISCUSSION TITLE
What's the intended way to draw dynamic text?

### DISCUSSION THREAD

**Discussion by MyNameIsTrez** [codam-coding-college/MLX42#50] [Q&A] [answered]
I want to draw an FPS counter in the corner of the program. I could call `mlx_put_string()`, delete it, and repeat this cycle every frame, but I'd rather not go with this approach. `mlx_put_string()` calls `mlx_draw_char()`, but since it is static I am not able to call it from my project. It might be that calling `mlx_draw_char()` on the same image wouldn't even overwrite the previous characters.  ¯\\\_(ツ)\_/¯

**Comment by W2Wizard** [comment, marked as answer]
The first approach is the naive way of doing it.

The intented way here is for you to write your own character drawing implementation.
You would use `mlx_put_string()` to init the string image and then your own character drawing function update the string if necessary.

A fancy implementation would also approriately resize the string image.
